src/main.rs

```diff
@@ -0,0 +1,116 @@
+use std::{
+    env, fs,
+    io::{stderr, Write},
+    path::Path,
+    process,
+};
+
+use getopts::Options;
+use libc::{c_int, getchar, putchar};
```

Comment:
Are there not Rust-native alternatives for get/putchar?

Reply:
I think I'd have to convert things to a UTF-8 string, where BF only deals in 8 bit bytes. I'm happy to be proven wrong though!

Reply:
A quick google doesn't turn anything up, so I can live with the FFI calls.